PoshSSDTBuildDeploy/Functions/InstallMicrosoftDataToolsMsBuild.ps1

```diff
@@ -26,8 +25,7 @@ function Install-MicrosoftDataToolsMSBuild {
     $nugetInstallMsbuild = "&$NugetExe install Microsoft.Data.Tools.Msbuild -ExcludeVersion -OutputDirectory $WorkingFolder"
     if ($DataToolsMsBuildPackageVersion) {
         if ($DataToolsMsBuildPackageVersion -lt "10.0.61026") {
-            Write-Error "Lower versions than 10.0.61026 will NOT work with Publish-DatabaseDeployment. For more information, read the post https://blogs.msdn.microsoft.com/ssdt/2016/10/20/sql-server-data-tools-16-5-release/"
-            Throw
+            Throw "Lower versions than 10.0.61026 will NOT work with Publish-DatabaseDeployment. For more information, read the post https://blogs.msdn.microsoft.com/ssdt/2016/10/20/sql-server-data-tools-16-5-release/"            
```

Comment:
Combined this so that it throws the exception with the error message and then the test can test the specific exception